src/finiteVolume/processes/twoPhaseEulerAdvection.cpp

```diff
@@ -191,6 +193,7 @@ PetscErrorCode ablate::finiteVolume::processes::TwoPhaseEulerAdvection::FormJaco
 }
 
 ablate::finiteVolume::processes::TwoPhaseEulerAdvection::TwoPhaseEulerAdvection(std::shared_ptr<eos::EOS> eosGas, std::shared_ptr<eos::EOS> eosLiquid,
```

Comment:
I would pass the two phase equation of state into this instance instead of the separate separate equation of states.  This way the same equation of state can be used in other functions/classes